src/Core/Models/GraphQLFilterParsers.cs

```diff
@@ -616,7 +633,8 @@ public static class FieldFilterParser
         IInputField argumentSchema,
         Column column,
         List<ObjectFieldNode> fields,
-        Func<object, string?, string> processLiterals)
+        Func<object, string?, string> processLiterals,
+        bool isListType = false)
```

Comment:
Add in the param summary as to what this is and how this helps in deciding between using LIKE/NOT_LIKE Vs ARRAY_CONTAINS/NOT_CONTAINS.

Reply:
I have mentioned the purpose of this flag. I think code is straightforward and explaining how things are changing with this flag.